pkg/dataplane/util/ansibleee.go

```diff
@@ -230,10 +230,14 @@ func labelsForOpenStackAnsibleEE(name string, labels map[string]string) map[stri
 }
 
 func (a *EEJob) addEnvFrom(job *batchv1.Job) {
```

Comment:
This method is not even used atm. I had a PR commit [1] already in the queue that was making  the change in a different context (i.e removing the interface). Anyway, I'll rebase that. 

[1] https://github.com/openstack-k8s-operators/openstack-operator/commit/64e9614a84192420cb8b18fa59ebf6a7130fd275